### DISCUSSION TITLE
List of Followers of Organization.

### DISCUSSION THREAD

**Discussion by rafidalhaque** [community/community#19051] [Other Feature Feedback, Questions, & Ideas] [answered]
Is this feature in roadmap to see the list of followers who are following my organization?

**Comment by ApidriuC** [comment]
Puedes encontrar más información de como visualizar los equipos y roles de la organización en el siguiente enlace:

https://docs.github.com/es/organizations/organizing-members-into-teams/about-teams

Allí explican donde se pueden ver los miembros y seguidores de la organización, si a eso te refieres.

**Reply by rafidalhaque** [reply]
Sorry, perhaps I did not explain my question well. 
I own a github organization. I want to see the users list who are following my organization. That's it...

**Comment by felipezanardi** [comment]
Unfortunately, don't have any way to see it.

**Comment by rilysh** [comment]
Currently GitHub haven't this feature to view followers for organisations, right now it's not possible to count followers for it. I guess, it's not even in their road map about it.

**Comment by melvincwng** [comment, marked as answer]
@rafidalhaque, unfortunately I don't think there is a way at the moment to check who are the followers of your organization.

Hope this clarifies, thanks!

**Reply by melvincwng** [reply]
@rafidalhaque, if this reply has helped you, please kindly 'Mark it as answer' so that other people are aware that this question has been addressed.

Thanks 😊

**Reply by rafidalhaque** [reply]
Marked 🌹. But if GitHub add this, it will be enjoyfull for me 🌸

**Comment by davidecarpini** [comment]
I don't get why after 2 years this important list is still not shown

**Reply by simonkovtyk** [reply]
True, even one year later...
